phac-nml/irida-next
# Feature: Group Samples View

Fixes #71. 

## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR adds a new view that displays all the samples for a group to help with the implementations of sharing samples and associating projects. The group samples view has a table displaying the sample and project names. 

Note: Table, headers, links, pagination, etc are included in this PR. CSS will be address in a future PR.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/phac-nml/irida-next/assets/325703/d1b24e5b-5c4c-4f6d-8333-bccca0603062)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._
1. Login as admin.
2. Go to a group.
3. Click on the new menu item "Samples".
4. You should see the new group samples view.
